libraries/AP_BoardConfig/AP_BoardConfig.cpp

```diff
@@ -355,11 +355,11 @@ void AP_BoardConfig::init_safety()
 /*
   notify user of a fatal startup error related to available sensors. 
 */
-bool AP_BoardConfig::_in_sensor_config_error;
+bool AP_BoardConfig::_in_config_error;
 
-void AP_BoardConfig::config_error(const char *fmt, ...)
+void AP_BoardConfig::throw_error(const char *err_type, const char *fmt, va_list arg)
 {
-    _in_sensor_config_error = true;
+    _in_config_error = true;
```

Comment:
Well, it's closer - but still wrong post-your-refactoring :-)

I don't have a better name, so *shrug*.

I *guess* it could be `_in_throw_error_loop`

Reply:
`_in_error_loop` ?